modules/installation-configure-proxy.adoc

```diff
@@ -44,6 +44,9 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-vmc-user-infra"]
 :vmc:
 endif::[]
+ifeval::["{context}" == "installing-openstack-installer-custom"]
```

Comment:
@MaysaMacedo Do you want this note to appear _only_ in [this document](https://deploy-preview-36621--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom?utm_source=github&utm_campaign=bot_dp), or should it appear in others, too?

Reply:
@maxwelldb it should appear only in the openstack cluster-wide proxy related, so yeah I believe only that one

Reply:
@MaysaMacedo Currently, this module is included in the doc that's linked in my previous comments, as well as this one: https://deploy-preview-36621--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-kuryr.html#installation-configure-proxy_installing-openstack-installer-kuryr

The `ifeval` configuration here means that your note _only_ appears in the linked document. 

Do you want it to appear:

- As it is right now
- In both docs
- In the latter doc (Kuryr) only

Reply:
@maxwelldb it fit better being only in Kuryr